ql-qlf-plugin/pmgen/pmgen.py

```diff
@@ -0,0 +1,797 @@
+#!/usr/bin/env python3
```

Comment:
This file is part of Yosys and certainly should not be checked into this repo. Since it is not installed along with Yosys and is needed for the build I'm not entirely sure how to proceed. Maybe add downloading it using wget directly from YosysHQ GitHub in the `pre-build` stage?